sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CasesView.java

```diff
@@ -812,7 +812,8 @@ public HorizontalLayout createStatusFilterBar() {
 									}));
 						}
 
-						if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.EVENT_SURVEILLANCE)) {
+						if (FacadeProvider.getFeatureConfigurationFacade().isFeatureEnabled(FeatureType.EVENT_SURVEILLANCE)
```

Comment:
in the `UiUtil` class there are som function to make these kind of checks shorter and cleaner, please use those ones and create new ones if needed